akamai_cloudlet_manager.gemspec

```diff
@@ -13,12 +13,11 @@ Gem::Specification.new do |s|
                    "lib/akamai_cloudlet_manager/policy_version.rb",
                    "lib/akamai_cloudlet_manager/policy.rb"
                  ]
-  s.executables << 'ac'
+  s.executables << 'acm'
   s.homepage    = 'https://github.com/gouravtiwari/akamai-cloudlet-manager'
   s.license       = 'MIT'
   s.add_runtime_dependency 'akamai-edgegrid', '~> 1.0', '>= 1.0.6'
   s.add_runtime_dependency 'json', '~> 2.1'
   s.add_runtime_dependency 'thor', '~> 0.20.0'
   s.add_development_dependency 'rspec', '~> 3.7'
-  s.add_development_dependency 'rake'
```

Comment:
I think rake is needed by travis, if we can keep it.

Reply:
my bad.